src/Symfony/Component/DependencyInjection/Attribute/AsAlias.php

```diff
@@ -20,7 +20,14 @@
 final class AsAlias
 {
     public function __construct(
+        /**
+         * The id of the service to alias.
```

Comment:
This is actually the id of the alias service.
The current description makes me think that this is the id that the alias will target but this is not the case (the target of the alias will be the service defined for the class on which the attribute is used)